examples/disaggregated/slurm/disagg_profiler/job_manager.py

```
@@ -0,0 +1,1107 @@
+        unset_env = ''
+        if 'unset_env' in config:
+            for key in config['unset_env']:
+                unset_env += f" -u {key}"
+        nsys = False
+        nsys_prefix = ''
+        if 'nsys' in config:
+            nsys = True
+        if nsys:
+            envs += " TLLM_PROFILE_RECORD_GC=1"
+            envs += " TLLM_NVTX_DEBUG=1"
+            if 'TLLM_PROFILE_START_STOP' in config['nsys']:
```

Ruff (0.12.2)

Line too long (222 > 120)

(E501)